### ISSUE TITLE
Paragraph tags randomly don't work

### ISSUE THREAD

**Issue by architarious** [wvuweb/cleanslate-cms#177]
In multiple browsers I keep having cleanslate not wanting to apply paragraph tags through the "block format" dropdown menu. Instead, i normally have to paste in the content, open the html editor and insert the tags manually. Sometimes I'll even put the tags in through html editor and it will erase them when I hit "save and replace". Both of these seem to be happening completely randomly in both firefox and chrome. 

Also, this is unrelated and I didn't think it merited it's own post, but is there a reason why it keeps wrapping everything with DIV tags?

**Comment by adamjohnson** [comment]
Many of the things you have specified deal with the HTML5 `contentEditable` attribute that [Mercury](https://jejacks0n.github.io/mercury/) (our editor) uses. The strange behavior and crappy markup that you’re seeing stems from lack of any standards around that attribute and different implementations by browser vendors.

The guardian team is documenting these here:

https://github.com/guardian/scribe/blob/master/BROWSERINCONSISTENCIES.md

Long story short, these bugs are due to a lack of clear, defined standards from browsers & their implementors. We’re considering workarounds (#2), but the best thing to do to get these fixed would be to contribute on these open source projects (like [Mercury](https://github.com/jejacks0n/mercury) and [Scribe](https://github.com/guardian/scribe)) and file/star bugs with browser vendors.

**Comment by architarious** [comment]
cool, thanks for the info
